packages/dendron-viz/package.json

```diff
@@ -25,14 +25,34 @@
     "clean": "rimraf lib && rimraf coverage",
     "prepublishOnly": "yarn build",
     "prebuild": "yarn clean",
-    "build": "yarn compile",
-    "buildCI": "yarn compile",
-    "compile": "tsc -p tsconfig.build.json "
+    "build": "yarn copyNonTSFiles && yarn compile",
+    "buildCI": "yarn copyNonTSFiles && yarn compile",
+    "copyNonTSFiles": "mkdir -p lib && cp ./src/loadModule.js lib/",
+    "compile": "tsc -p tsconfig.build.json",
+    "watch": "yarn copyNonTSFiles && yarn compile --watch"
   },
   "dependencies": {
+    "@actions/artifact": "^0.5.2",
```

Comment:
you probably don't need the `@actions/*` dependency. the github action will be in a separate package when we get to it

Reply:
Removed the use of @actions/core from `dendron.tsx`